torchaudio/backend/common.py

```diff
@@ -38,3 +38,14 @@ def __init__(
         self.num_channels = num_channels
         self.bits_per_sample = bits_per_sample
         self.encoding = encoding
+
+    def __str__(self):
+        return (
+            f"{self.__class__.__name__}("
```

Comment:
Although this is correct and more generic, TorchScript compiler has restriction in the kind of attributes it can access, and `self.__class__` is not accessible. Can you hard-code it?

Reply:
Thanks for the tip. I commited the necessary change.